packages/multi-select-combo-box/src/vaadin-multi-select-combo-box-scroller.js

```diff
@@ -17,6 +17,19 @@ class MultiSelectComboBoxScroller extends ComboBoxScroller {
     return 'vaadin-multi-select-combo-box-scroller';
```

Comment:
When I focus the dev page element, NVDA announces `"Element Combobox collapsed, required, invalid entry, has autocomplete, editable, blank"`, even though it has a pre-set value

Reply:
Seems that the "Please fill in this field." tooltip (intended for empty required native input) also gets shown.

<img width="295" alt="Screenshot 2022-04-28 at 10 23 32" src="https://user-images.githubusercontent.com/1222264/165699476-730a03a6-cdc5-4b0c-a1b5-94fe3ecf3680.png">

Reply:
@tomivirkki Updated the PR to use `aria-required="true"` instead of `required` as we discussed.
This removes the "Please fill in this field" tooltip. as well as "invalid entry" being announced.